server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/CustomMethodContract.scala

```diff
@@ -154,6 +174,31 @@ class CustomMethodModule extends AbstractModule {
     Multibinder.newSetBinder(binder(), classOf[Method])
       .addBinding()
       .to(classOf[CustomMethod])
+    Multibinder.newSetBinder(binder(), classOf[TypeName])
+      .addBinding()
+      .toInstance(CustomTypeName)
+    Multibinder.newSetBinder(binder(), classOf[GuiceProbe])
+      .addBinding()
+      .to(classOf[JmapEventBusProbe])
+  }
+}
+
+class TypeNameModule extends AbstractModule {
```

Comment:
This TypeName module should be part of `RFC8621MethodsModule` ? (At least the generic parts?)